### PR TITLE
update: readme with description to sdk

### PR DESCRIPTION
**Pull request by mhrynenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,113 @@
-# ldif-sdk
-Tools for parsing with LDIF files containing PKI certificates
+# LDIF SDK
+## Description
+
+General toolkit to work with ICAO CSCA MasterLists. It may:
+ * Read and parse into different data structures `.ldif` files (as an example [ICAO PKD](https://pkddownloadsg.icao.int/) was used)
+ * Create dynamic Merkle tree with treap data structure
+ * Hash underlying certificates public keys
+ * Build Merkle tree from previous point that stores certificates list and verifies inclusion of a certificate
```
Comment (violog) [inline README.md]: certificates' hashes list is better

File: README.md [hunk 2]
```diff
@@ -1,2 +1,113 @@
...
+        return errors.Wrap(err, "failed to create new ldif converter")
+    }
+```
+
+After reading and parsing LDIF data these certificates can be converted into different formats: 
+
+* PEM - using `converter.ToPem()` will reproduce an array of strings that stores certificates in a [PEM](https://datatracker.ietf.org/doc/html/rfc7468) format
+* X509 - using `converter.ToX509()` witll return an array of certificates in a [x509](https://datatracker.ietf.org/doc/html/rfc5280) format 
+
+In additional, there is a method `converter.RawPubKeys()` that gives an ability to get all public keys from parsed certificates, except duplicates and unsupported types (
```
Comment (violog) [inline README.md]:
```suggestion
In addition, there is a method `converter.RawPubKeys()` that gives an ability to get all public keys from parsed certificates, except duplicates and unsupported types (
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,113 @@
...
+nowadays it handles only RSA public keys ).
```
Comment (violog) [inline README.md]:
```suggestion
nowadays it handles only RSA public keys).
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,113 @@
...
+
+Previous functions returns new instance of a certificate tree that has several useful method to work with created data 
+structure:
+* `Root()` - get current tree root
+* `IsExists()` - check if the underlying treap tree is initialised (or whether the root is empty) 
+* `GenerateInclusionProof(pemCertificate)` - generates inclusion proof for given certificate. The proof is such structure:
+```go
+    type Proof struct {
+        // Existence indicates whether this is a proof of existence or non-existence.
+        Existence bool `json:"existence"`
```
Comment (violog) [inline README.md]: Existence and Order are now irrelevant

File: README.md [hunk 5]
```diff
@@ -1,2 +1,113 @@
...
+        return errors.Wrap(err, "failed to create new ldif converter")
+    }
+	
+    ...
+
+    converter, err := NewLDIF(rawBytes) // Parse raw file bytes 
+    if err != nil {
+        return errors.Wrap(err, "failed to create new ldif converter")
+    }
+```
```
Comment (violog) [inline README.md:47]: maybe, add a reference to our unit tests for more examples

Comment (mhrynenko) [reply README.md:47]: This reference is on the 57 line

**Review by violog — APPROVED** [review]

**Pull request merged**